unit-test/cs_compute_tests.c

```diff
@@ -841,7 +857,54 @@ void CS_ComputeApp_Test_Nominal(void)
 
     /* Sets AppInfo.CodeSize = 5, sets AppInfo.CodeAddress = 1, AppInfo.AddressesAreValid = true, and returns
      * CFE_SUCCESS */
-    UT_SetHandlerFunction(UT_KEY(CFE_ES_GetAppInfo), CS_COMPUTE_TEST_CFE_ES_GetAppInfoHandler1, NULL);
+    UT_SetHandlerFunction(UT_KEY(CFE_ES_GetModuleInfo), CS_COMPUTE_TEST_CFE_ES_GetModuleInfoHandler1, NULL);
+
+    /* Set to fail condition "NewChecksumValue != ResultsEntry -> ComparisonValue" */
+    UT_SetDeferredRetcode(UT_KEY(CFE_ES_CalculateCRC), 1, 2);
+
+    /* Execute the function being tested */
+    Result = CS_ComputeApp(&ResultsEntry, &ComputedCSValue, &DoneWithEntry);
+
+    /* Verify results */
+    UtAssert_True(DoneWithEntry == true, "DoneWithEntry == true");
+
+    UtAssert_True(ResultsEntry.NumBytesToChecksum == 5, "ResultsEntry.NumBytesToChecksum == 5");
+    UtAssert_True(ResultsEntry.StartAddress == 1, "ResultsEntry.StartAddress == 1");
+
+    UtAssert_True(ComputedCSValue == 2, "ComputedCSValue == 2");
+    UtAssert_True(ResultsEntry.ByteOffset == 0, "ResultsEntry.ByteOffset == 0");
+    UtAssert_True(ResultsEntry.TempChecksumValue == 0, "ResultsEntry.TempChecksumValue == 0");
```

Comment:
Avoid UtAssert_True in any new/updated code.  Use the macros that generate the string for you, for example UtAssert_UINT32_EQ (LT/GT/NE/etc).  There's ones for bools, strings, addresses, etc.  Only use the BOOL ones on boolean variables, not as part of a conditional.

Reply:
Thanks, I didn't know about these. I went in and removed the UtAssert_Trues for the other tests in the file as well.